modules/repository-s3/src/main/plugin-metadata/entitlement-policy.yaml

```diff
@@ -0,0 +1,4 @@
+ALL-UNNAMED:
+  - network:
+      actions:
+        - connect
```

Comment:
Looks like plugins don't need `accept` or `listen` after all?

Does the server even need those? I wonder if we could open our ports before init phase 2 is over? Then we wouldn't need to deal with `accpet` or `listen` at all.

Reply:
We are still missing NIO, which will require more permission/entitlements to more places/policies